apps/desktop/e2e/base.spec.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Example Playwright script for Electron
+ * showing/testing various API features
+ * in both renderer and main processes
+ */
+
+import {
+  expect,
+  test,
+  ElectronApplication,
+  Page,
+  _electron as electron,
+} from '@playwright/test';
+import { findLatestBuild, parseElectronApp } from 'electron-playwright-helpers';
+
+let electronApp: ElectronApplication;
+
+test.beforeAll(async () => {
+  // find the latest build in the out directory
+  const latestBuild = findLatestBuild('release/1.0.0');
+  // parse the directory and find paths and other info
+  const appInfo = parseElectronApp(latestBuild);
+  // set the CI environment variable to true
+  process.env.CI = 'e2e';
+  electronApp = await electron.launch({
+    args: [appInfo.main],
+    executablePath: appInfo.executable,
+  });
+  electronApp.on('window', async page => {
+    const filename = page.url().split('/').pop();
+    console.log(`Window opened: ${filename}`);
+
+    // capture errors
+    page.on('pageerror', error => {
+      console.error(error);
+    });
+    // capture console messages
+    page.on('console', msg => {
+      console.log(msg.text());
+    });
+  });
+});
+
+test.afterAll(async () => {
+  await electronApp.close();
+});
+
+let screen: Page;
+
+test.beforeEach(async () => {
+  const splash = await electronApp.firstWindow();
+  await splash.waitForEvent('close');
+  screen = await electronApp.firstWindow();
+});
+
+test('record', async () => {
+  await screen.pause();
+});
+
+test('check the title of window', async () => {
+  const title = await screen.title();
+  expect(title).toBe('Cypherock CySync');
+});
+
+test('Device connection screen', async () => {
+  const newwindow = screen.getByRole('heading', {
+    name: 'Your X1 Vault will now be authenticated through Cypherock to check its authenticity...(?)',
+  });
+  await newwindow.waitFor();
+  expect(newwindow).toBeVisible();
+});
+
+test('Device authentication successful', async () => {
+  const newwindow = screen.getByRole('heading', {
+    name: 'Your X1 Vault is successfully authenticated',
+  });
+  await newwindow.waitFor();
+  expect(newwindow).toBeVisible();
+});
+
+// test('Device auth fail due to internet disconnection', async () => {
+//
+// })
+
+test('Device disconnection during device auth', async () => {
+  await screen
+    .getByRole('heading', {
+      name: 'Your X1 Vault will now be authenticated through Cypherock to check its authenticity...(?)',
+    })
+    .waitFor({ timeout: 300000 });
+  const newwindow = screen.getByRole('heading', {
```

Comment:
Use pascal case for variables. Example `newWindow`.

Reply:
changed to camelCase